src/test/examples/create-id-model/generated/classes.ts

```diff
@@ -1,22 +1,14 @@
 // This file has been auto-generated by Warthog.  Do not update directly as it
 // will be re-written.  If you need to change this file, update models or add
 // new TypeGraphQL objects
-import { GraphQLID as ID } from "graphql";
 import {
   ArgsType,
   Field as TypeGraphQLField,
-  Float,
-  InputType as TypeGraphQLInputType,
-  Int
+  InputType as TypeGraphQLInputType
 } from "type-graphql";
 import { registerEnumType } from "type-graphql";
 
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const { GraphQLJSONObject } = require("graphql-type-json");
-
 import { BaseWhereInput, PaginationArgs } from "../../../../";
-// @ts-ignore
-import { CreateIDTest } from "../../create-id-model";
```

Comment:
Did all of this get removed by the code generator?

Reply:
This file doesn't appear to be auto-generated (not really sure how it got generated in the first place).  These changes were manual.  Happy to fix, though, if you can point me to where it actually comes from.